app/src/test/java/io/gnosis/safe/ui/transactions/TransactionListViewModelTest.kt

```diff
@@ -2,20 +2,42 @@ package io.gnosis.safe.ui.transactions
 
```

Comment:
We made quite extensive changes to this view model, why is this not reflected in the tests? (i.e. we call updateLocalTxLatest and added new mappings)

Reply:
added tests